### ISSUE TITLE
Unable to reproduce Cifar-10 results for WRN-28-10

### ISSUE THREAD

**Issue by humza909** [ildoonet/pytorch-randaugment#32]
Hi,

I am unable to reproduce cifar-10 results for the model WRN-28-10. The accuracy I get stays as low as 96.7% but the reported accuracy is 97.4% (this repo) / 97.3% (in paper). Anyone else facing similar issues? @ildoonet Is your reported accuracy the average of multiple runs or only for a single run?   

I am using the following command:

`python RandAugment/train.py -c confs/wresnet28x10_cifar10_b256.yaml`

Thanks

**Comment by dahuaxiya** [comment]
I get 96.7% test accuracy too after 200 epochs. Do you know why now?
